osmaxx-py/osmaxx/excerptexport/models/excerpt.py

```diff
@@ -4,7 +4,7 @@
 
 
 class Excerpt(models.Model):
-    name = models.CharField(max_length=128, verbose_name=_('name'))
+    name = models.CharField(max_length=128, verbose_name=_('name'), blank=False)
```

Comment:
This was probably my hint that lead to this change: `blank=False` [is the default](https://docs.djangoproject.com/en/1.9/ref/models/fields/#django.db.models.Field.blank), so can be left out here.